roles/openstack-setup/meta/main.yml

```diff
@@ -1,3 +1,4 @@
 ---
 dependencies:
   - role: endpoints
+  - role: nova-common
```

Comment:
This will cause nova-common tasks to be executed.

We should instead define this in openstack-meta/defaults/main.yml add that as dependency here. openstack-meta is light weight role to only define openstack common variables.